### DISCUSSION TITLE
questions not issues ;)

### DISCUSSION THREAD

**Discussion by rolikeusch** [stackernews/booger#2] [General]
I have 2 question, not issues... hope it's ok that i place those also here.

1. I saw that there is sqlite.js file, but I can't find something about this in the docs. is this an upcoming feature that i can also use a sqlite db? would be great!
2. in the plugs docs i saw that you use similar approach like strfry. I would like to use the whitelist script that you can find [here](https://github.com/hoytech/strfry/blob/master/docs/plugins.md), will this work without any modification? (sorry I'm unfortunately not a programmer :()

btw, very interesting relay!!

**Comment by huumn** [comment]
Hey! 

> I have 2 question, not issues... hope it's ok that i place those also here.

All good! I moved it to a discussion.

> 1. I saw that there is sqlite.js file, but I can't find something about this in the docs. is this an upcoming feature that i can also use a sqlite db? would be great!

That sqlite is not for "note" storage. Booger *currently* only supports postgres for note storage but we would like to provide other note storage options.

>  2. in the plugs docs i saw that you use similar approach like strfry. I would like to use the whitelist script that you can find [here](https://github.com/hoytech/strfry/blob/master/docs/plugins.md), will this work without any modification? (sorry I'm unfortunately not a programmer :()

It will not. Our plugin system is quite different. But I could write an adapter for strfry plugins. Overall, great questions. I'll convert most of them to issues.

**Comment by rolikeusch** [comment]
great to hear that you plan to added other storage options.
ok, if this with the adapter is possible it would be absolutely great!
